users/shared/.config/claude/skills/creating-pull-requests/SKILL.md

```
@@ -1,89 +1,151 @@
+|------|-------|
+| `feat` | New feature |
+| `fix` | Bug fix |
+| `docs` | Documentation |
+| `refactor` | Code refactoring |
+| `test` | Test changes |
+| `chore` | Build/tools |
+
+**Breaking:** Add `!` → `feat!: ...` or `fix scope!: ...`
+
+**Examples:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)